src/test/java/it/eng/idsa/dataapp/web/rest/exceptions/DataAppExceptionHandlerTest.java

```diff
@@ -0,0 +1,103 @@
+package it.eng.idsa.dataapp.web.rest.exceptions;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.entity.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.test.util.ReflectionTestUtils;
+import de.fraunhofer.iais.eis.Message;
+import it.eng.idsa.dataapp.util.MessageUtil;
+
+import it.eng.idsa.multipart.util.UtilMessageService;
+
+class DataAppExceptionHandlerTest {
+
+	@InjectMocks
+	private DataAppExceptionHandler dataAppExceptionHandler;
+	@Mock
+	private MessageUtil messageUtil;
+	private Message header;
+	private String issuerConnector = "http://w3id.org/engrd/connector/";
+	private String httpConfig = "form";
+	@Mock
+	HttpEntity resultEntity;
+	private ContentType payloadContentType = ContentType.TEXT_PLAIN;
+
+	@BeforeEach
+	public void init() {
+
+		MockitoAnnotations.initMocks(this);
+		header = UtilMessageService.getArtifactRequestMessage();
+		ReflectionTestUtils.setField(dataAppExceptionHandler, "issuerConnector", issuerConnector);
+		ReflectionTestUtils.setField(dataAppExceptionHandler, "httpConfig", httpConfig);
+
+	}
+
+//	@Test
+//	public void testHandleBadParametersException() throws IOException {
+//		BadParametersException ex = new BadParametersException("test", header);
+//		when(messageUtil.createMultipartMessageForm(MultipartMessageProcessor.serializeToJsonLD(header), null,
+//				payloadContentType)).thenReturn(resultEntity);
+//		dataAppExceptionHandler.handleBadParametersException(ex, null);
+//		assertNotNull(ex);
+//
+//	}
+
+	@Test
```

Comment:
Can this one be transformed to parametrized test?

Reply:
Technically it can, but we wouldn't achieve anything with it, since we don't have a single entry method for exceptions, so I think that parametrized test wouldn't be useful here,